spec/controllers/host_controller_spec.rb

```diff
@@ -64,6 +64,14 @@
       expect(response.status).to eq(200)
     end
 
+    it 'returns an error when trying to edit a host that does not support editing' do
+      session[:host_items] = [h1.id, h2.id]
+
+      get :edit
+      expect(controller.send(:flash_errors?)).to be_truthy
+      expect(response.status).to eq(302)
```

Comment:
@Fryguy, @agrare, not sure if this is the best way to test. Open to suggestions.